packages/core/components/FileList/useFileAccessContextMenu.ts

```diff
@@ -19,6 +21,46 @@ export default function useFileAccessContextMenu(filters?: FileFilter[], onDismi
     const userSelectedApplications = useSelector(interaction.selectors.getUserSelectedApplications);
     const { executionEnvService } = useSelector(interaction.selectors.getPlatformDependentServices);
 
+    const [hasPlateBarCode, setHasPlateBarCode] = React.useState(false);
+    let plateUILink: string | undefined = undefined;
+
+    fileSelection.fetchAllDetails().then((fileDetails) => {
+        console.log(fileDetails);
+        const annotations = fileDetails[0].annotations;
+        const platebarcode = annotations.find((x) => x.name === "Plate Barcode");
+        if (platebarcode != undefined && platebarcode.values != undefined) {
```

Comment:
Simpler using [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)
```suggestion
        if (!platebarcode?.values) {
```